### ISSUE TITLE
CodeLens is missing if handler starts with a folder

### ISSUE THREAD

**Issue by abrooksv** [aws/aws-toolkit-vscode#661]
**Describe the bug**

**To Reproduce**
Given:
```
Resources:
  HelloWorldFunction:
    Type: AWS::Serverless::Function # More info about Function Resource: https://github.com/awslabs/serverless-application-model/blob/master/versions/2016-10-31.md#awsserverlessfunction
    Properties:
      CodeUri: hello-world/
      Handler: foo/app.lambdaHandler
      Runtime: nodejs10.x
```

shows no CodeLens on the handler

**Expected behavior**
There should be a code lens

**Comment by teknogeek0** [comment]
I ran into this as well and all new SAM apps do it this way by default now.

**Comment by awschristou** [comment]
Hi @teknogeek0 this issue should have been addressed by #764 (which helped with #757). If you're still seeing this behavior, I'm interested in knowing.

Can you please list the following:
* Toolkit version
* SAM CLI Version
* runtime
* sam template Resource fragment

Using Windows 10, Toolkit 1.4.0, SAM CLI 0.37.0, I created a new node 10.x SAM App, and was able to see CodeLenses, and run the application. I then moved my app.js file to a 'foo' subfolder, and was also able to see CodeLenses and run the application.
